release note for MTM-57675 (#1786)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-4-0-improved-the-appearance-of-the-single-sign-on-sso-login-button.md

---
date: ""

> title: Improved the appearance of the SSO login button

Reviewer: I think title should be `Fixed the appearance of the SSO login button` instead of `Improved the appearance of the SSO login button.` because its a bug and not improvement

> title: Fixed the appearance of the Single Sign-On (SSO) login button.

Reviewer 2: ```suggestion
title: Fixed the appearance of the SSO (Single Sign-On) login button
```

> title: Improved the appearance of the Single Sign-On (SSO) login button.

Reviewer 2: ```suggestion
title: Improved the appearance of the SSO login button
```

> product_area: Device management & connectivity

Reviewer: I think product_area should be Platform services

change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57675

> version: 1019.4.0

Reviewer: I see the fixed version is different in the ticket than 1019.4.0 Please recheck

---

> The appearance of the SSO login button is unified with the Cumulocity application styles.

Reviewer 2: ```suggestion
The appearance of the SSO (single-sign-on) login button has been unified with the Cumulocity IoT application styles.
```
